functions/package.json

```diff
@@ -1,15 +1,17 @@
 {
   "name": "functions",
   "scripts": {
-    "lint": "eslint --ext .js,.ts .",
-    "lint:fix": "eslint --ext .js . --fix",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
     "build": "tsc",
     "serve": "npm run build && firebase emulators:start --only functions",
     "shell": "npm run build && firebase functions:shell",
     "start": "npm run shell",
     "deploy": "firebase deploy --only functions",
     "logs": "firebase functions:log",
-    "test": "jest"
+    "test": "jest",
+    "test-watch": "jest --watch",
+    "test-coverage": "jest --coverage"
   },
   "engines": {
     "node": "14"
```

Comment:
How was this chosen? I see algolia has 
```
"engines": {
    "node": ">=15.0.0"
  }
```

But node `16` is not the current release yet so I'm wondering.

Reply:
It was auto-generated when you create the project with firebase.
I try to exchange for 12 to see but it won't work so I let it that way

Reply:
was it 14 or 
```
 "node": ">=14.0.0"
```

?

Reply:
14

Reply:
I think it is better, fireStore should upgrade do node 16 in octobre when it will become the current release so lets be accessible to that version already!